shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -744,18 +744,19 @@ static bool ClipBoundsContainsPlatformViewBoundingRect(const SkRect& clip_bounds
     int64_t platform_view_id = composition_order_[i];
     std::vector<std::shared_ptr<FlutterPlatformViewLayer>> layers = layer_map[platform_view_id];
     UIView* platform_view_root = root_views_[platform_view_id].get();
-
-    if (platform_view_root.superview != flutter_view) {
-      [flutter_view addSubview:platform_view_root];
-    }
+    //    if (platform_view_root.superview == flutter_view) {
+    //      [platform_view_root removeFromSuperview];
```

Comment:
@cyanglaz it turns out it's the `removeFromSuperview` call making the platform view not touchable. So now we simply `addSubview`, which just reorder the subview if already added.